index.js

```diff
@@ -102,7 +139,17 @@ console.log('hello world')
  * Example Output: "WHY AM I SCREAMING"
  */
 
-// TODO: ADD CODE HERE
+let str = "Ctd Rocks 🤘🏼!";
+
+stringToUpper = (str) => {
```

Comment:
When you define an arrow function, the variable name it's stored in needs a keyword:

```js
const stringToUpper = () => {}
```